tests/tracking/test_client.py

```diff
@@ -503,15 +503,15 @@ def test_get_databricks_runtime_nondb(mock_spark_session):
 
 
 def test_get_databricks_runtime_in_notebook(mock_spark_session):
-    with mock.patch("mlflow.utils.databricks_utils.is_in_databricks_notebook", return_value=True):
+    with mock.patch("mlflow.utils.databricks_utils.is_in_databricks_runtime", return_value=True):
         get_databricks_runtime()
         mock_spark_session.conf.get.assert_called_once_with(
             "spark.databricks.clusterUsageTags.sparkVersion", default=None
```

Comment:
I think we can remove this test because `get_databricks_runtime` no longer checks whether we're in a notebook or not.